df2gspread/gfiles.py

```diff
@@ -77,7 +77,8 @@ def get_worksheet(gc, gfile_id, wks_name, write_access=False):
             wks = spsh.worksheet(wks_name)
         else:
             if write_access == True:
-                wks = spsh.add_worksheet(wks_name, 1000, 100)
+                rows, cols = new_sheet_dimensions
+                wks = spsh.add_worksheet(wks_name, rows, cols)
```

Comment:
Not sure what you mean here. `*new_sheet_dimensions`? And what kind of checking? Do you mean checking to make sure someone doesn't put in a value too large?